influx/annotations.go

```diff
@@ -0,0 +1,265 @@
+package influx
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/influxdata/chronograf/id"
+
+	"github.com/influxdata/chronograf"
+)
+
+const (
+	// AllAnnotations returns all annotations from the chronograf database
+	AllAnnotations = `SELECT "start_time", "modified_time_ns", "text", "type", "id" FROM "chronograf"."autogen"."annotations" WHERE "deleted"=false AND time >= %dns and "start_time" <= %d ORDER BY time DESC`
+	// GetAnnotationID returns all annotations from the chronograf database where id is %s
+	GetAnnotationID = `SELECT "start_time", "modified_time_ns", "text", "type", "id" FROM "chronograf"."autogen"."annotations" WHERE "id"='%s' AND "deleted"=false ORDER BY time DESC`
+	// DefaultDB is chronograf.  Perhaps later we allow this to be changed
+	DefaultDB = "chronograf"
```

Comment:
Maybe a better name would be `AnnotationsDB` rather than `Default`

Reply:
Fixed in 010f67dc77528a30a25e216b4bafced5a5506c72